js/runner_test.go

```diff
@@ -765,7 +765,7 @@ func TestVURunInterruptDoesntPanic(t *testing.T) {
 					assert.Contains(t, vuErr.Error(), "context canceled")
 				}()
 				<-ch
-				time.Sleep(time.Microsecond * 1) // NOTE: increase this in case of problems ;)
+				time.Sleep(time.Millisecond * 1) // NOTE: increase this in case of problems ;)
```

Comment:
nit: Could we increase it by less than 1000x :)

Reply:
Looking at the test - this doesn't really matter, so it is probably okay. I was thinking it is one of those cases where we are trying to tease out a race, but we literally want for `RunOnce` to actually start.

Reply:
but yeah, I agree would be good to keep it as close to its original value as possible, ofc! 
although.. I also wonder, do we really care about 1 ms? haha